eslint.config.js

```diff
@@ -11,7 +11,7 @@ export default [
   },
   {
     languageOptions: {
-      ecmaVersion: 2022,
+      ecmaVersion: "latest",
```

Comment:
I had to specifically add this to the prompt. 
It was set to 2022 as part of my upgrading the config file from an older format.
This is probably an example of something that could hold back upgrades in other repos if they also have older config files that seemed to be working well enough so far.